music.js

```diff
@@ -1,17 +1,32 @@
 // Define the Artist constructor function here, with a 'name' property that can be set at instantiation,
 // and a 'songs' property that is an empty array to start
-function Artist() {
-
+function Artist(name) {
+  this.name = name;
+  this.songs = [];
 }
 
 // Define the Song constructor function here, with 'title' and 'artist' properties that can be set at instantiation,
 // and a 'playCount' property that is an empty array to start
-function Song() {
-
+function Song(title, artist) {
+  this.title = title;
+  this.artist = artist;
+  this.playcount = 0;
```

Comment:
Remember, when you're naming variables in JS that are multiple words, use camelCasing.